packages/wonder-blocks-pill/src/components/pill.tsx

```diff
@@ -243,10 +242,12 @@ const _generateColorStyles = (clickable: boolean, kind: PillKind) => {
     }
 
     const pressColor =
-        kind === "transparent"
-            ? semanticColor.status.neutral.background
-            : // NOTE(WB-1880): This will be simplified once we split this into Badge and Pill.
-              mix(tokens.color.offBlack32, backgroundColor);
+        kind === "transparent" || kind === "neutral"
+            ? tokens.color.offBlack16
```

Comment:
I'm surprised to see `semanticColor` replaced by `tokens.color.` Is that intentional?